mcs/build/rules.make

```diff
@@ -25,6 +25,10 @@ Q=$(if $(V),,@)
 Q_MCS=$(if $(V),,@echo "$(if $(MCS_MODE),MCS,CSC)     [$(intermediate)$(PROFILE)] $(notdir $(@))";)
 Q_AOT=$(if $(V),,@echo "AOT     [$(intermediate)$(PROFILE)] $(notdir $(@))";)
 
+MAKE_Q=$(if $(V),,-s)
+MAKE_V=$(if $(V),V=1,)
```

Comment:
These are redondant: if `V=1` or `-s` are defined when calling `make`, then it's going to be passed down with `$(MAKE)`